src/v2/layouts/MemberRegistration/MemberRegistrationForm.tsx

```diff
@@ -30,61 +34,74 @@ export const MemberRegistrationForm = () => {
 			<Form>
 				<StyledFormWrapper>
 					<StyledInputContainer>
-						<CustomInput label={USER_NAME} name="userName" type="text" />
+						<CustomInput label={USER_NAME} name="name" type="text" />
 					</StyledInputContainer>
 
 					<StyledInputContainer>
 						<CustomInput label={FATHER_HUSBAND_NAME} name="fatherName" type="text" />
 					</StyledInputContainer>
 					<StyledSelect>
-						<CustomInput
-							label={GENDER}
-							name="gender"
-							type="select"
-							dropdownOptions={gender(strings)}></CustomInput>
+						<StyledInputContainer>
+							<CustomInput
+								label={GENDER}
+								name="gender"
+								type="select"
+								dropdownOptions={gender(strings)}
+							/>
+						</StyledInputContainer>
 					</StyledSelect>
 
 					<StyledInputContainer>
-						<CustomInput label={AGE} name="age" type="number" />
+						<CustomInput label={AGE} name="age" type="text" />
 					</StyledInputContainer>
 
 					<StyledInputContainer>
-						<CustomInput label={MOBILE_NUMBER} name="mobileNumber" type="phone" />
+						<CustomInput label={MOBILE_NUMBER} name="mobile" type="phone" />
 					</StyledInputContainer>
 
 					<StyledInputContainer>
-						<CustomInput label={BUSINESS} name="business" type="text" />
+						<CustomInput
+							label={BUSINESS + '(optional)'}
```

Comment:
Get optional text also from strings json only.
```suggestion
							label={`${BUSINESS} ${OPTIONAL_LABEL}`}
```